src/main/java/it/gov/pagopa/ranking/dto/mapper/PageOnboardingRequests2RankingPageDTOMapper.java

```diff
@@ -0,0 +1,34 @@
+package it.gov.pagopa.ranking.dto.mapper;
+
+import it.gov.pagopa.ranking.dto.RankingPageDTO;
+import it.gov.pagopa.ranking.dto.RankingRequestsApiDTO;
+import it.gov.pagopa.ranking.model.InitiativeConfig;
+import it.gov.pagopa.ranking.model.OnboardingRankingRequests;
+import org.springframework.data.domain.Page;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class PageOnboardingRequests2RankingPageDTOMapper {
+
+    public RankingPageDTO apply(
+            Page<OnboardingRankingRequests> pageOnboardingRequests,
+            List<RankingRequestsApiDTO> content,
+            InitiativeConfig initiativeConfig) {
+        RankingPageDTO out = new RankingPageDTO();
+
+        out.setContent(content);
+        out.setPageNumber(pageOnboardingRequests.getNumber());
+        out.setPageSize(pageOnboardingRequests.getSize());
+        out.setTotalElements(pageOnboardingRequests.getTotalElements());
+        out.setTotalPages(pageOnboardingRequests.getTotalPages());
+        out.setRankingStatus(initiativeConfig.getRankingStatus());
+        out.setRankingPublishedTimeStamp(initiativeConfig.getRankingPublishedTimeStamp());
+        out.setRankingGeneratedTimeStamp(initiativeConfig.getRankingGeneratedTimeStamp());
+        out.setRankingFilePath(initiativeConfig.getRankingPathFile());
+        // TODO total OK and KO
```

Comment:
totalOk and totalKo will be fields (long type) stored inside the InitiativeConfig itself
You can already add these fields and use theme here